cockpit-starter-kit.spec.in

```diff
@@ -7,7 +7,7 @@ License: LGPLv2+
 Source0: https://github.com/cockpit-project/starter-kit/releases/download/%{version}/%{name}-%{version}.tar.xz
 Source1: https://github.com/cockpit-project/starter-kit/releases/download/%{version}/%{name}-node-%{version}.tar.xz
 BuildArch: noarch
-BuildRequires:  nodejs
+BuildRequires: nodejs >= 12
```

Comment:
This is actually satisfied as nodejs version 10 has an "1:" epoch, argh!

Reply:
OK, with the epoch the [failure](https://download.copr.fedorainfracloud.org/results/packit/cockpit-project-starter-kit-514/centos-stream-8-x86_64/02992613-cockpit-starter-kit/builder-live.log.gz) now looks as expected:
```
Error: 
 Problem: conflicting requests
  - package nodejs-1:12.20.1-1.module_el8.4.0+649+b0593ddf.x86_64 is filtered out by modular filtering
  - package nodejs-1:14.16.0-2.module_el8.4.0+716+f63d6d3d.x86_64 is filtered out by modular filtering
  - package nodejs-1:14.17.3-2.module_el8.5.0+861+a6fe2112.x86_64 is filtered out by modular filtering
  - package nodejs-1:14.17.5-1.module_el8.6.0+939+4802ccb9.x86_64 is filtered out by modular filtering
  - package nodejs-1:16.4.2-1.module_el8.5.0+900+545f34ef.x86_64 is filtered out by modular filtering
  - package nodejs-1:16.7.0-2.module_el8.5.0+917+002e3877.x86_64 is filtered out by modular filtering
  - package nodejs-1:16.8.0-1.module_el8.6.0+928+039f010b.x86_64 is filtered out by modular filtering
```